providers/apis/defi/ethmulticlient/multi_client.go

```diff
@@ -139,6 +149,7 @@ func (m *MultiRPCClient) BatchCallContext(ctx context.Context, batchElems []rpc.
 			r, ok := req[blockNumReqIndex].Result.(*string)
 			if !ok {
 				errs[i] = fmt.Errorf("result from eth_blockNumber was not a string")
+				results[i] = result{0, nil, errs[i]}
```

Comment:
i think we can just remove the `errs` slice now, looks like its duplicate data as the error is getting put in `results` now